cli/subcommands/src/cargo_hax.rs

```diff
@@ -66,25 +73,361 @@ fn rustflags() -> String {
     [rustflags, "--cfg hax".into()].join(" ")
 }
 
+const ENGINE_BINARY_NAME: &str = "hax-engine";
+const ENGINE_BINARY_NOT_FOUND: &str = const_format::formatcp!(
```

Comment:
A dependency for this seems a little unnecessary. Can we do without?

Reply:
The `const_format`? Sure, I can inline `hax-engine`, let me push that